Dockerfile

```
@@ -1,4 +1,8 @@
-
 FROM jetty:9-alpine
-COPY target/*.war /var/lib/jetty/webapps/ROOT.war
-RUN java -jar $JETTY_HOME/start.jar --create-startd --approve-all-licenses
+COPY target/*.war $JETTY_BASE/webapps/ROOT.war
+RUN java -jar $JETTY_HOME/start.jar \
+  --create-startd \
+  --approve-all-licenses \
+  --add-to-start=logging-logback \
+  --module=logging-logback \
+  -Dsystem.properties=io.jenkins.plugins
```

don't you want this?
```suggestion
  -Dstacktrace.app.packages=io.jenkins.plugins
```

---

its actually sentry.stacktrace.app.packages, but makes more sense in code than the docker file, so removed it.